docs/publishers/cli.md

```diff
@@ -0,0 +1,65 @@
+# DPM: the datapackage manger CLI
+
+The dpm is a command-line tool aimed to help publishers to prepare and upload data to the Datapackage Registry. With dpm you will be able to:
+
+* Validate your data to ensure its quality
+* Upload datapackage
+* Tag uploaded datapackage to create historical snapshot
+* Remove uploaded datapackage that is no longer needed
+
+## Installation
+
+Dpm is available on pypi [hopefully soon it will be], you can install it using pip in the terminal:
+```
+[sudo] pip install dpm
+```
+
+**NOTE**
+
+You can alternatively install unstable version directly from the code repository:
+
+```
+[sudo] pip install git+https://github.com/frictionlessdata/dpm-py.git
+```
```

Comment:
I'd start before configuration with running the help. Ultimately, the command line tool's most up to date docs and state is in its live form and help. Thus, I'd briefly say something like:

You can see the latest commands and get help by doing:

```
dpmpy -h
```

Below we give an overview of some of the main commands you can use.